packages/react-core-notifications/package.json

```diff
@@ -51,7 +51,7 @@
     "@types/react": "^17.0.2",
     "@types/react-test-renderer": "^17.0.1",
     "@types/testing-library__jest-dom": "^5.14.1",
-    "eslint": "^8.13.0",
+    "eslint": "^8.4.1",
```

Comment:
Why the downgrade?

Reply:
Tried to match the version used in `react-core` when I saw some eslint related issues. But it was unrelated to the version used; and was due to a typo in the eslintrc file name. Will revert back to use the latest version.